src/components/ChatList/ChannelListHeaderItem.tsx

```diff
@@ -3,6 +3,7 @@ import FastImage from 'react-native-fast-image';
 import {StyleSheet, Text, View} from 'react-native';
 
 import dimen from '../../utils/dimen';
+import {SIGNED} from '../../hooks/core/constant';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `SIGNED` from `../../hooks/core/constant` is a good change as it replaces the hardcoded string 'SIGNED' with a constant. This will help prevent bugs due to typos and make the code easier to maintain if the value needs to be changed in the future.
```diff
- import {colors} from '../../utils/colors';
+ import {SIGNED} from '../../hooks/core/constant';
  import {colors} from '../../utils/colors';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->